pkgs/top-level/stage.nix

```diff
@@ -269,6 +269,12 @@ let
     });
   };
 
+  # Auto-calls packages declared in the named package hierarchy in pkgs/unit
+  unitPackages = self: super:
+    lib.mapAttrs (name: file:
+      self.callPackage file { }
+    ) (import ./unit-files.nix);
```

Comment:
Off-topic: this snippet made me think a lot about how to format things in nixpkgs, especially since it has a function call with long first argument and short last argument. I remember you saying about how to use `flip` in these cases, but actually I think that using `<|` before the last argument would work even better. Yet another reason why I want `<|` in the language …